src/PotatoQuery.php

```diff
@@ -0,0 +1,268 @@
+use Elchroy\PotatoORMExceptions\FaultyOrNoTableException;
+use Elchroy\PotatoORMExceptions\NoRecordException;
+use PDO;
+use PDOException;
+
+class PotatoQuery
+{
+    /**
+     * [$connection The connection to be used to communicate with the database. To be instantiated during construction.].
+     *
+     * @var [type] PDO Connection.
+     */
+    public $connection;
+
+    /**
+     * Setup the connection to aid all queries.
+     *
+     * @param PotatoConnector|null $connector [description]
+     */
+    public function __construct(PotatoConnector $connector = null)
+    {
+        $connector = $connector == null ? new PotatoConnector() : $connector;
+        $this->connection = $connector->connection;
+    }
+
+    /**
+     * Get all records from the specified database table.
+     *
+     * @param string $table   The table name
+     * @param string $columns The columns to be gotten from the database table
+     *
+     * @return array An array of objects, each representing a record fetched from the database table.
+     */
+    public function getFrom($table, $columns = '*')
+    {
+        $sql = "SELECT $columns FROM $table";
+        $statement = $this->connection->prepare($sql);
+        $statement == false ? $this->throwFaultyOrNoTableException($table) : '';
+        $execution = $this->tryExecuting($statement);
+        $result = $statement->fetchAll(PDO::FETCH_CLASS);
+        $result == false ? $this->throwNoRecordException($table) : '';
+
+        return $result;
+    }
+
+    /**
+     * Get only one record from the datbase table, given the id of the record to be retrieved.
+     *
+     * @param string $table The database table name
+     * @param int    $id    The id of the recored to be fethced.
+     *
+     * @return model object        The record fetched form the database table as an object
+     */
+    public function getOne($table, $id)
+    {
+        $sql = "SELECT * FROM $table WHERE id = :id ";
+        $statement = $this->connection->prepare($sql);
+        $statement == false ? $this->throwFaultyOrNoTableException($table) : '';
+        $statement->bindParam(':id', $id);
+        $execution = $this->tryExecuting($statement);
+        $result = $statement->fetchObject($table);
+        $result == false ? $this->throwNoRecordException($table, $id) : '';
+
+        return $result;
+    }
+
+    /**
+     * Store a new recored inside athe database table.
+     *
+     * @param string $table The table into which it is required to add a new record
+     * @param string $data  An aray of the columns to be used for inseting into the database
+     *
+     * @return bool The result of the query execution. True if successful. False otherwise.
+     */
+    public function storeIn($table, $data)
+    {
+        $columnsString = $this->getColumns($data);
+        $count = (int) count($data);
+        $sql = "INSERT INTO $table $columnsString VALUES (".$this->putQuesMarks($count).')';
```

Comment:
Since `.` is an operator, you need to precede it with a space and also add a space after it, thus:
```"INSERT INTO $table $columnsString VALUES (" . $this->putQuesMarks($count) . ')'```